Controllers/TeamController.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Threading.Tasks;
+using CribblyBackend.Models;
+using CribblyBackend.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CribblyBackend.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class TeamController : ControllerBase
+    {
+        private readonly ITeamService TeamService;
+        public TeamController(ITeamService TeamService)
+        {
+            this.TeamService = TeamService;
```

Comment:
📖 🌵 use camelCase for private fields and method parameters

Reply:
fixed